values.yaml

```diff
@@ -444,3 +444,69 @@ airflow:
     password: password1
     port: 5432
     user: admin
+  service:
```

Comment:
airflow config seems to have been changed, the PR description should explain why and how